x-pack/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -31,11 +34,29 @@ export class HeadlessChromiumDriver {
 
   public async open(
     url: string,
-    { headers, waitForSelector }: { headers: Record<string, string>; waitForSelector: string }
+    {
+      conditionalHeaders,
+      waitForSelector,
+    }: { conditionalHeaders: ConditionalHeaders; waitForSelector: string }
   ) {
+
     this.logger.debug(`HeadlessChromiumDriver:opening url ${url}`);
+    await this.page.setRequestInterception(true);
+    this.page.on('request', (interceptedRequest: any) => {
+      if (this._shouldUseCustomHeaders(conditionalHeaders.conditions, interceptedRequest.url())) {
+        this.logger.debug(`Using custom headers for ${interceptedRequest.url()}`);
```

Comment:
Can we have it as:
- `Sending request to ${interceptedRequest.url()} with custom headers`
- `Sending request to ${interceptedRequest.url()} without custom headers`?

Minor / nitpicky, but it would give us a standard prefix to look for or grep on

Reply:
I personally find it easier to scan for the current messages, and we can still grep for "custom headers for" which is specific enough for my tastes.